packages/clay-core/src/tree-view/TreeViewItem.tsx

```diff
@@ -728,6 +728,7 @@ export function ItemStack({
 					<Button
 						aria-controls={`${item.key}`}
 						aria-expanded={expandedKeys.has(item.key)}
+						aria-labelledby={labelId}
```

Comment:
This needs to be revisited. This shouldn't be a button. It's just a visual indicator. We changed the expand / collapse interaction from this button to the parent container a while ago.

Reply:
@MarioLeandro sounds like a good plan.